tests/Unit/Evolution/Test_TagsDomain.cpp

```diff
@@ -151,25 +153,36 @@ void test() noexcept {
           initial_time,
           std::array<DataVector, 3>{{{0.0}, {velocity[2]}, {0.0}}});
 
-  const std::unique_ptr<
-      domain::CoordinateMapBase<Frame::Grid, Frame::Inertial, Dim>>
-      grid_to_inertial_map = std::make_unique<ConcreteMap<Dim>>(
-          create_coord_map<Dim>(functions_of_time_names));
+  using MapPtr = std::unique_ptr<
+      domain::CoordinateMapBase<Frame::Grid, Frame::Inertial, Dim>>;
+  const MapPtr grid_to_inertial_map =
+      IsTimeDependent ? MapPtr(std::make_unique<ConcreteMap<Dim>>(
+                            create_coord_map<Dim>(functions_of_time_names)))
+                      : MapPtr(std::make_unique<domain::CoordinateMap<
+                                   Frame::Grid, Frame::Inertial,
+                                   domain::CoordinateMaps::Identity<Dim>>>());
 
   const double time = 3.0;
   auto box = db::create<simple_tags, compute_tags>(
       time, mesh, grid_coords, element_to_grid_inverse_jacobian,
       std::move(functions_of_time), grid_to_inertial_map->get_clone());
 
   const auto check_helper = [&box, &mesh]() noexcept {
-    const boost::optional<Scalar<DataVector>>& div_frame_velocity =
-        db::get<domain::Tags::DivMeshVelocity>(box);
-    REQUIRE(static_cast<bool>(div_frame_velocity));
-    CHECK(*div_frame_velocity ==
-          divergence(
-              db::get<domain::Tags::MeshVelocity<Dim>>(box).get(), mesh,
-              db::get<domain::Tags::InverseJacobian<Dim, Frame::Logical,
-                                                    Frame::Inertial>>(box)));
+    if (IsTimeDependent) {
+      const boost::optional<Scalar<DataVector>>& div_frame_velocity =
+          db::get<domain::Tags::DivMeshVelocity>(box);
+      REQUIRE(static_cast<bool>(div_frame_velocity));
+      CHECK(*div_frame_velocity ==
+            divergence(
+                db::get<domain::Tags::MeshVelocity<Dim>>(box).get(), mesh,
+                db::get<domain::Tags::InverseJacobian<Dim, Frame::Logical,
+                                                      Frame::Inertial>>(box)));
+    } else {
+      // Check that in the time-independent case that the divergence of the mesh
```

Comment:
[nitpick] I suggest "In the time-independent case, check that the divergence..."